.gitmodules

```diff
@@ -1,3 +1,3 @@
-[submodule "crates/llama-cpp-bindings/llama.cpp"]
+[submodule "crates/llama-cpp-server/llama.cpp"]
```

Comment:
Seems to be a bug introduced by a previous PR? Tests were failing until making this change

Reply:
seems not rebase against main? https://github.com/TabbyML/tabby/blob/main/.gitmodules

Reply:
I did rebase against main